Sources/Datadog/FeaturesIntegration/WebView/WKUserContentController+Datadog.swift

```diff
@@ -0,0 +1,76 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+import WebKit
+
+// TODO: RUMM-1794 rename the method
+public extension WKUserContentController {
+    func addDatadogMessageHandler(allowedWebViewHosts: [String]) {
+        let bridgeName = DatadogMessageHandler.name
+
+        let messageHandler = DatadogMessageHandler(
+            eventBridge: DatadogEventBridge(
+                logEventConsumer: WebLogEventConsumer(),
+                rumEventConsumer: WebRUMEventConsumer()
+            )
+        )
+        add(messageHandler, name: bridgeName)
+
+        // WebKit installs message handlers with the given name format below
+        // We inject a user script to forward `window.{bridgeName}` to WebKit's format
+        let webkitMethodName = "window.webkit.messageHandlers.\(bridgeName).postMessage"
+        // `WKScriptMessageHandlerWithReply` returns `Promise` and `browser-sdk` expects immediate values.
+        // We inject a user script to return `allowedWebViewHosts` instead of using `WKScriptMessageHandlerWithReply`
+        let allowedWebViewHostsString = allowedWebViewHosts
+            .map { return "\"\($0)\"" }
+            .joined(separator: ",")
+
+        let js = """
+        window.\(bridgeName) = {
+          send(msg) {
+            \(webkitMethodName)(msg)
+          },
+          getAllowedWebViewHosts() {
+            return '[\(allowedWebViewHostsString)]'
```

Comment:
🐞 Here we pass user input into JS code without sanitization or escaping and it can clearly break the syntax. Seems like a good candidate for unit test 👌.